assignments/F-01-93/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
we dont needed a git ignore  here . I have already added one

Reply:
Ok I'll add these to that gitignore and delete this